pkg/remote/message.go

```diff
@@ -334,3 +366,12 @@ func FillSendMsgFromRecvMsg(recvMsg, sendMsg Message) {
 	sendMsg.SetProtocolInfo(recvMsg.ProtocolInfo())
 	sendMsg.SetPayloadCodec(recvMsg.PayloadCodec())
 }
+
+func CreateConcatKey(serviceName, methodName string) string {
```

Comment:
Not a good name, especially as a exported symbol.

`BuildMultiServiceKey` suggested.

Reply:
Btw, please remember to add comments to exported symbols.

Reply:
done